src/Confluent.Kafka/Consumer.cs

```diff
@@ -1140,10 +1156,14 @@ internal Consumer(ConsumerBuilder<TKey, TValue> builder)
         ///     The consume result.
         /// </returns>
         /// <remarks>
-        ///     OnPartitionsAssigned/Revoked and OnOffsetsCommitted events may
+        ///     The partitions assigned/revoked and offsets commited handlers may
         ///     be invoked as a side-effect of calling this method (on the same
         ///     thread).
         /// </remarks>
+        /// <exception cref="ConsumeException">
+        ///     Thrown if a consume request is unsuccessful for any reason. Inspect
```

Comment:
This makes it sound like there is a consume request sent to the broker, which only covers a subset of possible error cases.
Might want to rephrase as "Thrown when consumption fails..", "Thrown on consumer-level errors..", ..